content/v2.0/organizations/members/add-member.md

```diff
@@ -7,25 +7,33 @@ menu:
     name: Add a member
     parent: Manage members
 weight: 201
-draft: true
 ---
 
-Use the InfluxDB user interface (UI) or the `influx` command line interface (CLI)
-to add a member to an organization.
+Use the `influx` command line interface (CLI) to add a member to an organization.
 
-## Add a member to an organization in the InfluxDB UI
+<!-- ## Add a member to an organization in the InfluxDB UI -->
 
-1. Click the **Settings** tab in the navigation bar.
+<!-- 1. Click the **Settings** tab in the navigation bar. -->
 
-    {{< nav-icon "settings" >}}
+<!--     {{< nav-icon "settings" >}} -->
 
-2. Click on the name of an organization, then select the **Members** tab.
+<!-- 2. Click on the name of an organization, then select the **Members** tab. -->
 
-_Complete content coming soon_
+<!-- _Complete content coming soon_ -->
 
 ## Add a member to an organization using the influx CLI
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add) to add a user to an organization.
+You will need the ID of the user you are trying to add.
 
-_Complete content coming soon_
+1. Get a user's ID by running `influx org members list -n <org-name>`.
```

Comment:
1. This command lists existing members of the org. Users are created outside of an organization, then added as a member of an organization. You need to use the `influx user find` command to list users with their IDs, then use the user ID to add them as a member of an org.
2. Command examples should be in code blocks, not inline code.